docs/book/v3/cookbook/debug-toolbars.md

```diff
@@ -69,7 +69,7 @@ return array_merge(ConfigProvider::getConfig(), [
 ]);
 ```
 
-In addition, ensure the [PSR-15 HTTP message factory interfaces](https://www.php-fig.org/psr/psr-15/)
+In addition, ensure the [PSR-17 HTTP message factory interfaces](https://www.php-fig.org/psr/psr-15/)
```

Comment:
It should be https://www.php-fig.org/psr/psr-17/

Reply:
fixed.